вычисление заданных величин/вычисление заданных величин/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace вычисление_заданных_величин
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double a, b, c, n, k;
+            Console.WriteLine("введите стороны прямоугольника:");
+            a = double.Parse(Console.ReadLine());
+            b = double.Parse(Console.ReadLine());
+            n = 2 * (a + b);
+            Console.WriteLine("периметр прямоугольника:" + n);
+            k = a * b;
+            Console.WriteLine("площадь прямоугольника:" + k);
+            c = Math.Sqrt(a * a + b * b);
+            Console.WriteLine("длина диагонали прямоугольника:" + c);
```

Comment:
Не стоит злоупотреблять использованием локальных переменных для промежуточных вычислений.
Если какая-то величина нужна только один раз, может вычислить ее там, где она нужна? Конечно, во всем нужно соблюдать меру и не допускать в коде чересчур сложных и неудобочитаемых выражений. В этой ситуации дополнительные локальные переменные будут уместны.
```suggestion
            Console.WriteLine("периметр прямоугольника:" + 2 * (a + b));
            Console.WriteLine("площадь прямоугольника:" + a * b);
            Console.WriteLine("длина диагонали прямоугольника:" + Math.Sqrt(a * a + b * b)
```